meta-lmp-base/classes/kernel-lmp-fitimage.bbclass

```diff
@@ -72,26 +206,26 @@ fitimage_emit_section_config() {
 				if [ -n "${loadable_line}" ]; then
 					conf_desc="${conf_desc}${sep}loadables"
 				fi
-				loadable_line="${loadable_line}loadable_${i} = \"loadable@${LOADABLE}\"; "
+				loadable_line="${loadable_line}loadable_${i} = \"loadable${FIT_NODE_SEPARATOR}${LOADABLE}\"; "
 			fi
 		done
 	fi
 
 	if [ "${8}" = "1" ]; then
-		default_line="default = \"conf@${3}\";"
+		default_line="default = \"conf${FIT_NODE_SEPARATOR}${3}\";"
 	fi
 
 	cat << EOF >> ${1}
                 ${default_line}
-                conf@${3} {
+                conf${FIT_NODE_SEPARATOR}${3} {
                         description = "${8} ${conf_desc}";
                         ${kernel_line}
                         ${fdt_line}
                         ${ramdisk_line}
                         ${setup_line}
                         ${fpga_line}
                         ${loadable_line}
-                        hash@1 {
+                        hash-1 {
```

Comment:
could this be ${FIT_NODE_SEPARATOR} here instead of the hard coded - ?

This would be he same question for the rest of these occurances.

Reply:
There is no need to use that here because we're not manually referring to this node, so we can simply just move to - instead.

Now for dtb, kernel, initrd, etc this is a bit more complicated because we're referring to them manually in our boot scripts.